godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/variantTypeMapping.kt

```diff
@@ -0,0 +1,99 @@
+@file:Suppress("EXPERIMENTAL_API_USAGE")
+
+package godot.core
+
+import godot.Object
+import kotlin.reflect.KClass
+
+private fun asBooleanTypeConverter(variantToConvert: Variant) = variantToConvert.asBoolean()
```

Comment:
Why create that many new functions when you can directly return Variant::asBoolean in the map below ?

Reply:
that's a good question i have no answer for xD
I'll change it ^^